src/foam/parsers/html/lib.js

```diff
@@ -506,6 +506,12 @@
     'yen': '¥',
   };
 
+  var escapeKeys = Object.keys(escapes).map(function(key) {
```

Comment:
One last thing: Drop extra line between `escapeKeys` and `escapeSequencesRegExp`, and add a comment above the two:

```js
// FUTURE: Lazily instantiate RegExp to save memory.
```